sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -118,17 +122,17 @@
  *
  * <p>To configure a Kafka source, you must specify at the minimum Kafka <tt>bootstrapServers</tt>
  * and one or more topics to consume. The following example illustrates various options for
- * configuring the source :
+ * configuring the source:
  *
  * <pre>{@code
  *
  *  pipeline
  *    .apply(KafkaIO.<Long, String>read()
  *       .withBootstrapServers("broker_1:9092,broker_2:9092")
  *       .withTopic("my_topic")  // use withTopics(List<String>) to read from multiple topics.
- *       // set a Coder for Key and Value
- *       .withKeyCoder(BigEndianLongCoder.of())
- *       .withValueCoder(StringUtf8Coder.of())
+ *       .withKeyDeserializer(new LongDeserializer())
+ *       .withValueDeserializer(new StringDeserializer())
+ *
```

Comment:
Why? It's not the case for coders either. You supply a coder, rather than a coder class; makes sense to supply a deserializer, not deserializer class. There can be useful fields in a deserializer class in general.